### DISCUSSION TITLE
Warn - RATE LIMIT EXCEEDED - what do to ?

### DISCUSSION THREAD

**Discussion by undauntedjogging** [jef/streetmerchant#1609] [Help]
Hi guys,

sometimes i get the following error:
**[11:36:39] warn :: ✖ [saturn] [sony (sonyps5de)] ps5 digital :: RATE LIMIT EXCEEDED**

Does anyone know what to do here?

Page Timeout is in the config already to 60sec.
PAGE_TIMEOUT=60000

**Comment by jsauve** [comment]
Set `PAGE_SLEEP_MIN` to a higher value than default

**Reply by undauntedjogging** [reply]
Thanks, i will test it.
